huxley/core/migrations/0001_initial.py

```diff
@@ -252,8 +252,8 @@ def backwards(self, orm):
             'state': ('django.db.models.fields.CharField', [], {'max_length': '16'}),
             'times_attended': ('django.db.models.fields.PositiveSmallIntegerField', [], {'default': '0'}),
             'waitlist': ('django.db.models.fields.BooleanField', [], {'default': 'False'}),
-            'zip_code': ('django.db.models.fields.CharField', [], {'max_length': '16'})
+            'zip_code': ('django.db.models.fields.CharField', [], {'max_length': '16'}),
```

Comment:
@patilpranay, there should be a generated migration attached to this diff that adds the `assignments_finalized` field. You may have generated it but forgotten to add it to the commit?